website/docs/d/guardduty_detector.html.markdown

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Can you please add a sidebar link for this data source in `website/aws.erb`? Thanks!

Reply:
The sidebar link for the data source has been added.